console/src/view/task.rs

```diff
@@ -148,22 +148,27 @@ impl TaskView {
         ]);
 
         // Just preallocate capacity for ID, name, target, total, busy, and idle.
-        let mut metrics = Vec::with_capacity(6);
-        metrics.push(Spans::from(vec![
+        let mut overview = Vec::with_capacity(6);
```

Comment:
+1 for renaming this.

now that it also has a line for the location, we should probably update the `with_capacity` call:
```rust
        // Just preallocate capacity for ID, name, target, location, total, busy, and idle.
        let mut overview = Vec::with_capacity(7);
```